vite.config.ts

```diff
@@ -4,5 +4,5 @@ import react from '@vitejs/plugin-react'
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
-  base: '/pynorte2023-site/',
+  base: '/pynorte2023-site',
```

Comment:
Talvez isso possa quebrar no github-pages. Ele precisa que seja `/REPO/`

Veja a documentação: https://vitejs.dev/guide/static-deploy.html#gitlab-pages-and-gitlab-ci